docs/buildah-bud.md

```diff
@@ -336,11 +344,9 @@ that the PID namespace in which `buildah` itself is being run should be reused,
 or it can be the path to a PID namespace which is already in use by another
 process.
 
-**--platform**="Linux"
+**--platform**="OS/ARCH"
 
-This option has no effect on the build. Other container engines use this option
-to control the execution platform for the build (e.g., Windows, Linux) which is
-not required for Buildah as it supports only Linux.
+Set the OS/ARCH instead of the current operating system and architecture of the machine in the image manifest and config (for example `linux/arm`).
```

Comment:
I guess we should mention that --platform overrides --os/arch.

Reply:
Sure, mentioned that as well :)